package/package.sh

```diff
@@ -19,11 +19,13 @@ package_one=ON
 strip_enable="FALSE"
 usage="Usage: ${0} -v <version> -n <ON/OFF> -s <TRUE/FALSE> -b <BRANCH> -g <ON/OFF>"
 project_dir="$(cd "$(dirname "$0")" && pwd)/.."
-build_dir=${project_dir}/build
+modules_dir=${project_dir}/modules
+storage_dir=${modules_dir}/storage
```

Comment:
It will cause confusion when the `ENABLE_BUILD_STORAGE` is ON.

Reply:
graph will always need not to build storage, so it`s safe to clone storage here.

Reply:
if someone uses his code to use the package.sh, the storage will not clone again. and it use the same common.